SpatialGDK/Source/SpatialGDK/Public/SpatialCommonTypes.h

```diff
@@ -11,7 +11,9 @@
 // These are not a type of key supported by TMap.
 using Worker_EntityId_Key = int64;
 using Worker_RequestId_Key = int64;
+
```

Comment:
This is to make it clearer that the comment above doesn't apply to the declarations below